FixtureListScrapers/retrieve_ICC_fixtures.py

```diff
@@ -0,0 +1,28 @@
+#retrieve list of all ICC fixtures
+import requests
+from requests.structures import CaseInsensitiveDict
+import shutil
+import json
+
+headers = CaseInsensitiveDict()
+headers["Accept"] = "application/json"
+
+#retrieve first page, and find number of pages
+page = 0 #first page to download
+url = 'https://cricketapi-icc.pulselive.com/fixtures?pageSize=300&page=%s' % page  
```

Comment:
you want only completed matches, don't you?

Reply:
Yes, that was the intention, but I wasn't able to figure out how to do it without sending duplicate queries to the API; ended up deciding to just filter that information when processing in R. I see you've figured out an alternate solution in the meantime.